site/content/docs/5.2/layout/gutters.md

```diff
@@ -18,29 +18,29 @@ toc: true
 
 `.gx-*` classes can be used to control the horizontal gutter widths. The `.container` or `.container-fluid` parent may need to be adjusted if larger gutters are used too to avoid unwanted overflow, using a matching padding utility. For example, in the following example we've increased the padding with `.px-4`:
 
-{{< example >}}
+{{< example class="bd-example-row" >}}
```

Comment:
IMO there's an issue in this page when using `.bd-example-row` that makes disappear the space between column that is useful to understand the features presented in this page.